video/profiles.go

```diff
@@ -134,7 +142,10 @@ func GetDefaultPlaybackProfiles(video InputTrack) ([]EncodedProfile, error) {
 		// check it here.
 		lowerQualityThanSrc := profile.Height < video.Height && profile.Bitrate < video.Bitrate
 		if lowerQualityThanSrc {
-			relativeBitrate := float64(profile.Width*profile.Height) * (float64(videoBitrate) / float64(video.Width*video.Height))
+			// relativeBitrate needs to be slightly higher than the proportional average bitrate of the source video.
+			// Livepeer network uses bitrate to set max bitrate for encoding, so for the video to look good, we multiply
+			// it by a factor of 1.2.
+			relativeBitrate := 1.2 * float64(profile.Width*profile.Height) * (float64(videoBitrate) / float64(video.Width*video.Height))
```

Comment:
how did we determine the 1.2 multiplier?

Reply:
I didn't really. I just know that what we currently have is too low, because we're setting: `max bitrate = source avg bitrate` which gives poor results.

As described in the doc, I plan to not set it at all, when the Os start to support the `quality` param (Phase 2).